packages/frontend/src/components/common/ResourceView/ResourceViewGrid/index.tsx

```diff
@@ -59,47 +78,80 @@ const ResourceViewGrid: FC<ResourceViewGridProps> = ({
                             {group.name}
                         </Text>
                     )}
-
-                    <SimpleGrid cols={3} spacing="lg">
-                        {group.items.map((item) => (
-                            <Anchor
-                                component={Link}
-                                to={getResourceUrl(projectUuid, item)}
-                                key={item.type + '-' + item.data.uuid}
-                                sx={{
-                                    display: 'block',
-                                    color: 'unset',
-                                    ':hover': {
-                                        color: 'unset',
-                                        textDecoration: 'unset',
-                                    },
-                                }}
-                            >
-                                {item.type === ResourceViewItemType.SPACE ? (
-                                    <ResourceViewGridSpaceItem
-                                        item={item}
-                                        onAction={onAction}
-                                    />
-                                ) : item.type ===
-                                  ResourceViewItemType.DASHBOARD ? (
-                                    <ResourceViewGridDashboardItem
-                                        item={item}
-                                        onAction={onAction}
-                                    />
-                                ) : item.type === ResourceViewItemType.CHART ? (
-                                    <ResourceViewGridChartItem
-                                        item={item}
-                                        onAction={onAction}
-                                    />
-                                ) : (
-                                    assertUnreachable(
-                                        item,
-                                        `Resource type not supported`,
-                                    )
-                                )}
-                            </Anchor>
-                        ))}
-                    </SimpleGrid>
+                    <DragDropContext onDragEnd={handleOnDragEnd}>
+                        <Droppable
+                            droppableId="pinned-charts"
+                            isDropDisabled={isReorderDisabled}
```

Comment:
if you want to merge it you should probably pass-in `false` here otherwise it will be enabled for pinned items

[PinnedItemsPanel/index.tsx](https://github.com/lightdash/lightdash/pull/5049/files?diff=unified&w=1#diff-25e99326c09c59d0564e23ef6f40269aaa64cbd322463b79fea535750e2823e7)